rules/windows/registry_event/sysmon_runonce_persistence.yml

```diff
@@ -0,0 +1,25 @@
+title: Run Once task configuration in registry - Persistence method
+id: c74d7efc-8826-45d9-b8bb-f04fac9e4eff
+description: Rule to detect the configuration of Run Once registry key. Configured payload can be run by runonce.exe /AlternateShellStartup
+author: Avneet Singh @v3t0_
+status: experimental
+date: 2020/10/15
+references:
+    - https://twitter.com/pabraeken/status/990717080805789697
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSBinaries/Runonce.yml
+tags:
+    - attack.execute
+    - attack.t1218
```

Comment:
Just a check, you want to reference execution? Because T1218 is related to defense evasion

Reply:
yes, there is a mismatch. I'll modify the yaml file to reflect the right ones. Thanks a lot @w0rk3r